backend/src/core/common_schema.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum
+
+from pydantic import BaseModel
+
+
+class QueryParams(BaseModel):
+    page: int = 1
+    per_page: int = 25
+
+    @property
+    def offset(self) -> int:
+        return (self.page - 1) * self.per_page
+
+    @property
+    def limit(self) -> int:
+        return self.per_page
```

Comment:
Im not sure if property works with pydantic

Reply:
Works, i tested this and it's fine